app/helpers/application_helper/button/pdf.rb

```diff
@@ -2,4 +2,8 @@ class ApplicationHelper::Button::Pdf < ApplicationHelper::Button::Basic
   def visible?
     PdfGenerator.available?
   end
+
+  def check_rbac?
+    %w(render_report_pdf chargeback_download_pdf compare_download_pdf).include?(self[:child_id]) ? role_allows?(:feature => self[:child_id]) : true
```

Comment:
fyi @martinpovolny we (@PanSpagetka, @romanblanco and me) decided to rename ```check_rbac?``` to ```role_allows_feature?``` as the ```check_rbac?``` is not a proper naming for its functionality